packaging/Dockerfile.essential

```diff
@@ -0,0 +1,52 @@
+ARG DPDK_VERSION=18.05
+ARG OS_VER=18.10
+FROM ubuntu:${OS_VER}
+
+RUN apt-get update && apt-get install -y \
+    g++-5 \
+    gcc-5 \
```

Comment:
👍 @fbalicchia Anyway you can generate a PR with this included? I completely missed your changes earlier and committed another docker file.